core/src/table/table.module.css

```diff
@@ -51,3 +51,53 @@ in native HTML */
 .container tr:last-child td {
 	border-bottom-style: none;
 }
+
+/* Cell spacing */
+
+/* Medium */
+.medium th,
+.medium td {
+	padding: 12px 8px;
+}
+
+.medium td:first-child,
+.medium th:first-child {
+	padding-left: 16px;
+}
+
+.medium td:last-child,
+.medium th:last-child {
+	padding-right: 16px;
+}
+
+.small th,
+.small td {
+	padding: 8px 6px;
+}
+
+/* Small */
+.small td:first-child,
+.small th:first-child {
+	padding-left: 12px;
+}
+
+.small td:last-child,
+.small th:last-child {
+	padding-right: 12px;
+}
+
+/* Large */
+.large th,
+.large td {
+	padding: 16px 12px;
+}
+
+.large td:first-child,
+.large th:first-child {
+	padding-left: 16px;
+}
+
+.large td:last-child,
+.large th:last-child {
+	padding-right: 16px;
+}
```

Comment:
This is not correct. This should be double the size at line 92 (so 24px)

Actually I think we should only change the vertical spacing, not horizontal... it'd be great if you can fix that, or else I'll have another PR

Reply:
Updated, sorry for the inconvenient because I haven't worked with CSS for a quite long time 😢

Reply:
No worry :)) I would help every contributor line by line and couldn't be happier